### ISSUE TITLE
feature request: add support for styles

### ISSUE THREAD

**Issue by hansemannn** [tmcw/togeojson#19]
Hey there,

happy to see this project is still maintained - thank you for that!

One question I'd like to raise is if there are plans to map the .gpx styles to the .geojson format as well. The so-called "[Geojson CSS](https://wiki.openstreetmap.org/wiki/Geojson_CSS)" markup is used for that and could be applied from the `gpxx:DisplayColor` GPX attribute. I could contribute that change if the general idea is approved by you.

Thanks!

**EDIT**: It seems like basic styles are already supported [here](https://github.com/tmcw/togeojson/blob/master/lib/gpx.js#L11-L26) but it seems limited to certain styles. The latest Garmin routes have a format as following:
```xml
  <rte>
    <name>27.05.2020 Tour 5_357 km_zyanblau</name>
    <extensions>
      <gpxx:RouteExtension>
        <gpxx:IsAutoNamed>false</gpxx:IsAutoNamed>
        <gpxx:DisplayColor>Cyan</gpxx:DisplayColor>
      </gpxx:RouteExtension>
      <trp:Trip>
        <trp:TransportationMode>Motorcycling</trp:TransportationMode>
      </trp:Trip>
    </extensions>
    ...
  </rte>
```

**Comment by tmcw** [comment]
Hey Hans!

Sure thing, adding support for these Garmin extensions sounds like a good idea. I'd prefer not to implement GeoJSON CSS, though. I haven't seen any implementations of that in the wild, and the spec itself has some rather bad design decisions - for example, putting `className` on the Feature object rather than nesting it under properties. That kind of thing will just make compatibility with strict GeoJSON parsers and converters harder.

Instead, something like `gpxx-isautonamed` under the properties object seems fine to me, a more verbatim translation of what the GPX attribute is saying.

**Comment by hansemannn** [comment]
Interesting thought. Maybe even map them all automatically, e.g.:
```json
{
  "properties": {
    ...
    "gpxx-isautonamed": false,
    "gpxx-displaycolor": "Cyan"
  }
}
```
By simply detecting the prefix and mapping the nodes to JSON keys. Or should we rather constraint it? Thanks for your feedback!

**Comment by tmcw** [comment]
Yep, I think automatically mapping values under RouteExtension would work!

**Comment by tmcw** [comment]
Fixed in #20!
